l10n/Makefile

```diff
@@ -4,7 +4,7 @@ NODE_BINDIR = ../node_modules/.bin
 export PATH := $(NODE_BINDIR):$(PATH)
 
 # Where to find apps (it can be multiple paths).
-APPS =  ../apps/files
+APPS =  ../apps/files ../apps/pdf-viewer ../apps/markdown-editor
```

Comment:
we need to replace the whole makefile in near future...

Reply:
as long as the root Makefile stays I'm fine with this. Our sync process requires make l10n-*